File: app/src/main/java/com/neome/api/ent/agent/sig/SigAgentEntUserImport.java
// neome.ai API - do not change
//

package com.neome.api.ent.agent.sig;

import com.neome.api.meta.base.Types.HandleKey;
import com.neome.api.meta.base.dto.EnvValidationError;
import com.neome.api.nucleus.base.sig.Sig;

import java.util.Map;

@SuppressWarnings("unused")
public class SigAgentEntUserImport extends Sig
{
  public Map<HandleKey, EnvValidationError> errorMap;
}

File: app/src/main/java/com/neome/api/ent/agent/sig/SigAgentEntUserImport.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.agent.sig

import com.neome.api.meta.base.dto.EnvValidationError
import com.neome.api.meta.base.Types.HandleKey
import java.util.Map
import com.neome.api.nucleus.base.sig.Sig

interface SigAgentEntUserImport : Sig
{
  val errorMap: Map<HandleKey, EnvValidationError>
}